### DISCUSSION TITLE
[PSD7] Exercise 2.23

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#153] [Esercizi (A-L)]
![image](https://user-images.githubusercontent.com/69389461/210017553-a14e6e73-514e-4d96-b58a-f8ae887fd77a.png)

**Comment by Elia-Belli** [comment]
Da dimostrare: $\overline{A + B + C} = \bar A \bar B \bar C$
#### Induzione Perfetta
Con L indico l'output del termine a sinistra e con R l'output del termine a destra, se gli output corrispondono alla è dimostrata l'uguaglianza.

**A B C** | **L R**
0 0 0 | 1 1
0 0 1 | 0 0
0 1 0 | 0 0
0 1 1 | 0 0
1 0 0 | 0 0
1 0 1 | 0 0
1 1 0 | 0 0
1 1 1 | 0 0

(analogo nella forma duale)

**Comment by fraxb04** [comment]
nella tabella della verità è errato mettere tutti 1 dal secondo termine in poi, dato che nell'or qualsiasi termine sommato ad 1 da 1, e negandolo otteniamo 0, analogamente per l'and qualsiasi numero moltiplicato per 0 da 0, dato che devono essere tutti 1 per avere 1

**Comment by Elia-Belli** [comment]
@fraxb04 hai ragione, corretto

**Comment by Nw4pS** [comment]
@fraxb04 ma la prima equazione scritta da @Elia-Belli non è negata quindi quando A=1 B=0 o C=1 è sempre 1
